app/src/app/navtitle/navtitle.component.sass

```diff
@@ -12,6 +12,7 @@
   left: 0
   width: 100%
   height: $navbarheight
+  z-index: 999
```

Comment:
999 is huge! I can't image a page with 1000 layers. Please use a lower and "rounder" value like 5, 10 or 20 (which already is a lot).